src/com/google/javascript/jscomp/Compiler.java

```diff
@@ -1318,6 +1318,40 @@ Node parseInputs() {
         processAMDAndCommonJSModules();
       }
 
+      if (options.lowerFromEs6() || options.transformAMDToCJSModules ||
```

Comment:
"||" should be on a new line according to https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break